packages/shared/components/tiles/tileContents/DelegationActivityTileContent.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+    import { localize } from '@core/i18n'
+    import { DelegationActivity, getActivityTileTitle } from '@core/wallet'
+    import { truncateString } from '@core/utils'
+    import { ActivityTileContent } from '@ui'
+    import { Icon } from '@lib/auxiliary/icon'
+
+    export let activity: DelegationActivity
+
+    $: action = localize(getActivityTileTitle(activity))
+    $: formattedAsset = {
+        text: truncateString(activity.validatorAddress, 6, 6),
```

Comment:
I think it is better to use the account address of the one that created the delegation, as is the case with the rest of the activities